packages/web/src/scss/components/Toast/README.md

```diff
@@ -217,84 +244,155 @@ Use our JavaScript plugin to open a Toast **that is present in the DOM,** e.g.:
+          aria-expanded="true"
+        >
+          <svg width="24" height="24" aria-hidden="true">
+            <use xlink:href="/icons/svg/sprite.svg#close" />
+          </svg>
+          <span class="accessibility-hidden">Close</span>
+        </button>
       </div>
-      <button
-        type="button"
-        class="Button Button--small Button--square Button--inverted"
-        data-spirit-dismiss="toast"
-        aria-controls="my-dismissible-toast"
-        aria-expanded="true"
-      >
-        <svg width="24" height="24" aria-hidden="true">
-          <use xlink:href="/icons/svg/sprite.svg#close" />
-        </svg>
-        <span class="accessibility-hidden">Close</span>
-      </button>
     </div>
     <!-- ToastBar: end -->
   </div>
 </div>
 <!-- Toast: end -->
 ```
 
+### Creating Dynamic ToastBars
+
+To create ToastBar components dynamically, make sure you have the `data-spirit-element="toast-queue"` attribute set on
+the `.Toast__queue` element, with just the ToastBar template inside the [`<template>`][mdn-template] tag.
+The `<template>` tag must be inserted anywhere inside the Toast container. Our JavaScript Toast plugin will then pick up
+the template and apply it on any toasts to be shown to the user, using the configuration provided.
+
+```html
+<div id="toast-example" class="Toast Toast--bottom Toast--center" role="log">
+  <div class="Toast__queue" data-spirit-element="toast-queue">
+    <!-- This is the template for new ToastBar components: -->
+    <template data-spirit-snippet="item">
+      <div class="ToastBar is-hidden" data-spirit-color="inverted" data-spirit-populate-field="item">
+        <div class="ToastBar__box">
+          <div class="ToastBar__content">
+            <svg width="20" height="20" aria-hidden="true" data-spirit-populate-field="icon">
+              <use xlink:href="/icons/svg/sprite.svg#info" />
+            </svg>
+            <div class="ToastBar__message" data-spirit-populate-field="message"></div>
+          </div>
+          <button
+            type="button"
+            class="Button Button--small Button--square"
+            data-spirit-color="inverted"
+            data-spirit-populate-field="close-button"
+            data-spirit-dismiss="toast"
+            aria-expanded="true"
+          >
+            <svg width="24" height="24" aria-hidden="true">
+              <use xlink:href="/icons/svg/sprite.svg#close" />
+            </svg>
+            <span class="accessibility-hidden">Close</span>
+          </button>
+        </div>
+      </div>
+    </template>
+  </div>
+</div>
+```
+
+Then configure and create a new Toast instance and call the `show` method on it, for example:
+
+```js
+import Toast from '@lmc-eu/spirit-web/dist/js/Toast';
+
+const toast = new Toast(null, {
+  color: 'informative', // One of ['inverted' (default), 'success', 'warning, 'danger', 'informative']
+  containerId: 'toast-example', // Must match the ID of the Toast container in HTML
```

Comment:
Since some configuration properties are required to be properly configured I think it will be suitable to create here some kind of options table with those properties. Also, keep in mind that the config can be used the same way as Bootstrap's one, so the properties can be passed by `data-spirit-config="", see https://getbootstrap.com/docs/5.3/components/toasts/#options

🤦 I know, this is a little bit of undocumented magic :-)

Reply:
Well, I was almost unaware of the `data-spirit-config` option. Anyway, I don't think it's suitable for our use case because the toasts are constructed by JavaScript from a `<template>` in HTML. At the first glance, I don't see a way how to do that since we don't yet have the HTML when we are configuring the toast…

If you feel we should get back to this, please file an issue for it. Thank you!

Reply:
Ok.